dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/JdbcSelectTask.java

```diff
@@ -17,25 +17,34 @@
 package com.google.edwmigration.dumper.application.dumper.task;
 
 import com.google.common.io.ByteSink;
+import com.google.edwmigration.dumper.application.dumper.connector.ZonedInterval;
 import com.google.edwmigration.dumper.application.dumper.handle.JdbcHandle;
 import java.sql.Connection;
 import java.sql.SQLException;
+import java.util.Optional;
 import javax.annotation.Nonnull;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.jdbc.core.ResultSetExtractor;
 
 /** @author shevek */
-public class JdbcSelectTask extends AbstractJdbcTask<Void> {
+public class JdbcSelectTask extends AbstractJdbcTask<Summary> {
 
   @SuppressWarnings("UnusedVariable")
   private static final Logger LOG = LoggerFactory.getLogger(JdbcSelectTask.class);
 
   @Nonnull private final String sql;
+  private final Optional<ZonedInterval> interval;
 
-  public JdbcSelectTask(@Nonnull String targetPath, @Nonnull String sql) {
+  public JdbcSelectTask(
+      @Nonnull String targetPath, @Nonnull String sql, Optional<ZonedInterval> interval) {
     super(targetPath);
     this.sql = sql;
+    this.interval = interval;
```

Comment:
I'm not sure `interval` should belong to this task, maybe we can have a `JdbcSelectIntervalTask`? That would remove the need for `Optional.empty()`.